tools/cli/FOAS.json

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
what's the purpose of this file? can we find a folder to it?

Reply:
I removed it